src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java

```diff
@@ -190,6 +199,31 @@
  *         the very same way as it is done by HTTPS. If "false" the host name is not verified at all. Per default the
  *         hostname is verified (true).
  *     </li>
+ *
+ *     <li>
+ *         <b>ch.hevs.cloudio.endpoint.token</b><br>
+ *
+ *         This property contains a single-use authentication token for the endpoint. This token is used to retrieve
```

Comment:
This property contains a single-use authentication token for endpoint provisioning. This token is used to retrieve certificates and settings using the cloud.iO Rest API.